apps/calcom/config.json

```diff
@@ -0,0 +1,106 @@
+    "form_fields": [
+        {
+            "type": "password",
+            "label": "Next.js Auth secret. Generate one with `openssl rand -base64 32`",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "CALCOM_NEXTAUTH_SECRET"
+        },
+        {
+            "type": "password",
+            "label": "Calendso Encryption Key. Encryption key that will be used to encrypt CalDAV credentials, choose a random string, for example with `dd if=/dev/urandom bs=1K count=1 | md5sum`",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "CALENDSO_ENCRYPTION_KEY"
+        },
+        {
+            "type": "text",
+            "label": "MS Graph Client ID. Used for the Office 365 / Outlook.com Calendar integration",
+            "required": false,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "MS_GRAPH_CLIENT_ID"
+        },
+        {
+            "type": "password",
+            "label": "MS Graph Client Secret. Used for the Office 365 / Outlook.com Calendar integration",
+            "required": false,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "MS_GRAPH_CLIENT_SECRET"
+        },
+        {
+            "type": "text",
+            "label": "Zoom Client ID. Used for the Zoom integration",
+            "required": false,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "ZOOM_CLIENT_ID"
+        },
+        {
+            "type": "password",
+            "label": "Zoom Client Secret. Used for the Zoom integration",
+            "required": false,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "ZOOM_CLIENT_SECRET"
+        },
+        {
+            "type": "email",
+            "label": "Configures the global From: header whilst sending emails.",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "EMAIL_FROM"
+        },
+        {
+            "type": "fqdn",
+            "label": "Email server host (SMTP)",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "EMAIL_SERVER_HOST"
+        },
+        {
+            "type": "number",
+            "label": "Email server port (SMTP)",
+            "required": true,
+            "min": 1,
+            "max": 8,
+            "env_variable": "EMAIL_SERVER_PORT"
+        },
+        {
+            "type": "password",
+            "label": "Email server password (SMTP)",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "EMAIL_SERVER_PASSWORD"
+        },
+        {
+            "type": "text",
+            "label": "Node.js environment type (production, development, test)",
+            "required": true,
+            "min": 1,
+            "max": 1024,
+            "env_variable": "CALCOM_NODE_ENV"
```

Comment:
I suggest to not make this an option and hardcode it to `production` in the `docker-compose.yml`